### ISSUE TITLE
Minecraft consistently times out

### ISSUE THREAD

**Issue by ThePsionic** [sonicsnes/node-gamedig#89]
Both `minecraft` and `minecraftping` time out on 1.13 servers. Attempted with 5 retries, 10,000 ms per socket timeout and 100,000 ms overall. Gives the error message `UDP watchdog timeout` or `TCP watchdog timeout` respectively.

**Comment by exwizzard** [comment]
I have tested this on latest vanilla 1.13.1 and its working fine.

**Comment by sonicsnes** [comment]
If you're still having this issue, give us a server IP which is exhibiting the issue :)

**Comment by ThePsionic** [comment]
Code:
```
const Gamedig = require("gamedig");

Gamedig.query({
	type: 'minecraft',
    host: 'minecraft.hollowworld.co.uk',
    maxAttempts: 5,
    socketTimeout: 10000,
    attemptTimeout: 100000
}).then((state) => {
	console.log(state);
}).catch((error) => {
	console.log(error);
});
```
Hollowworld (the IP in the above code) is a Spigot 1.12.2 server. `mc.sylph.tech` is a vanilla whitelisted 1.13.1 server. I have no idea what version `mc.hypixel.net` is on nowadays, but that server also doesn't work. All of these have been confirmed to work using Dinnerbone's server querier (however, his query python script repo is outdated and also does not work from my local machine).

I have tested all of the above servers with `minecraftping` and they seem to work now - it's just `minecraft` that's still giving me constant `UDP watchdog timeout` errors.

**Comment by sonicsnes** [comment]
The `minecraft` protocol returns the entire player list, and I believe the server doesn't do any packet splitting, which means it tries to put the entire player list into a single UDP packet (which obviously is over the byte limit if there are more than a a hundred or so players online).

There's nothing we can do if the server sends us back an invalid packet, since it would be dropped on their end. `minecraftping` is an alternative which does not include the player list, and thus is not affected by the packet size limit.

Does dinnerbone's include a list of player names?

**Comment by sonicsnes** [comment]
Looks like `minecraftping` actually includes basically everything these days. Perhaps we should just alias `minecraft` to `minecraftping` and move the old implementation to `minecraft-legacy`?

**Comment by ThePsionic** [comment]
As far as I can see, the only discernible difference between `minecraft` and `minecraftping` (according to ["""official""" docs](https://wiki.vg/Server_List_Ping)) is that the former also returns the base64 representation of the favicon of the server.

**Comment by Wqrld** [comment]
Bump, can you add favicon support to minecraftping? the old module seems to be completely broken

**Comment by sonicsnes** [comment]
I played around with this a bit more today, and it appears that both modes work exactly as they should.

The `minecraft` protocol only works if the server has queries enabled, and there are few enough players online that the response can fit in a single udp packet. This minecraft issue is documented in [MC-104861](https://bugs.mojang.com/browse/MC-104861)

This worked for me: `bin/gamedig.js --type minecraft --host play.skyblocks.org --debug`

The `minecraftping` protocol works all the time, but minecraft itself does not supply player names or the favicon using this protocol.

**Comment by sonicsnes** [comment]
Fixed in 1.0.47 by 9b9277172dc6011bafdc12fe2b99b262129adf28

**Comment by sonicsnes** [comment]
@Wqrld In 1.0.47, you should be able to access the favicon from within the state.raw object.
